review/get_review_ids.py

```diff
@@ -15,8 +15,8 @@
 import random
 from datetime import datetime
 
-import crum
 import pytz
+import crum
```

Comment:
I think the actual problem here is that the package fails to declare its dependency on [django-crum](https://pypi.org/project/django-crum/); this only works because there are no tests here, and the code is run in edx-platform which has that installed.  Could you add both `django-crum` and `pytz` to `install_requires` in setup.py?  You'll probably have to revert the import order change here.